src/chain/index.ts

```diff
@@ -75,11 +76,11 @@ export class BeaconChain extends EventEmitter {
 
```

Comment:
not part of this PR but `isValidBlock` could be reduced to one line

Reply:
isValidBlock is still a WIP, but it's intended to be this part of the fork choice rule spec:

 > For a beacon block, block, to be processed by a node, the following conditions must be met:
> *   The parent block with root block.previous_block_root has been processed and accepted.
> *   An Ethereum 1.0 block pointed to by the state.latest_eth1_data.block_hash has been processed and accepted.
> *   The node's Unix time is greater than or equal to state.genesis_time + block.slot * SECONDS_PER_SLOT.

https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_fork-choice.md#beacon-chain-processing